edit/cms_functions.php

```diff
@@ -0,0 +1,459 @@
+<?php
+
+/*
+ * Gets a PDO object to use for database queries. This PDO's fetch mode is set to FETCH_ASSOC by default.
+ *
+ * @Return returns the PDO object to use for our given queries.
+ */
+function getPDO() {
+    $db = new PDO('mysql:dbname=CMS;host=127.0.0.1','root');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+/*
+ * Displays the form for editing the About Me section of the portfolio. The value for each input is filled automatically with the current entry
+ * in the database.
+ *
+ * @return Returns the input elements for the About Me form, formatted as HTML. This should be placed inside a form element, before the submit
+ * element.
+ */
+function displayAboutMeInput() {
+    $stmt = getPDO()->prepare('SELECT `name`,`title`,`desc` FROM `about` LIMIT 1');
+    $stmt->execute();
+    $array = $stmt->fetch();
+    $output = '<div class="longinput"><label>Name: </label><input name="name" type="text" value="' . $array['name'] . '"></div>';
+    $output .= '<div class="longinput"><label>Title: </label><input name="title" type="text" value="' . $array['title'] . '"></div>';
+    $output .= '<div class="longinput"><label>Desc: </label><input name="desc" type="text" value="' . $array['desc'] . '"></div>';
```

Comment:
this could be a text area rather that an text input

Reply:
changed in 3258689